SCP/SCP.cpp

```diff
@@ -37,13 +36,14 @@ time_t print_current_time();
 int main(int argument_count, char * argv[])
 {
 	if (!valid_arguments(argument_count, argv)) {
-		return 0;
+		//return 0;
```

Comment:
sorry - its fine to turn back on - I commented out while stuffing about last night - I'll fix it now